src/controllers/http-api/v1/info-http-api-controller-v1.js

```diff
@@ -0,0 +1,33 @@
+import { createRequire } from 'module';
+import BaseController from '../base-http-api-controller.js';
+
+const require = createRequire(import.meta.url);
+const { version } = require('../../../../package.json');
+
+class InfoControllerV1 extends BaseController {
+    constructor(ctx) {
+        super(ctx);
+
+        this.blockchainModuleManager = ctx.blockchainModuleManager;
+    }
+
+    handleRequest(_, res) {
+        this.returnResponse(res, 200, {
+            version,
+            ...this.filterConfig(),
+        });
+    }
+
+    filterConfig() {
+        const nodeConfigData = {
+            tripleStores: Object.entries(this.config.modules.tripleStore.implementation)
```

Comment:
Why can't we use getImplementationNames for triple stores, just overwrite it in tripleStore module manager

Reply:
We can, but this route was added just for testing purposes, it can be removed